content/learn.md

```diff
@@ -81,6 +81,8 @@ We highly recommend pairing R4DS with the [RStudio cheatsheets](https://www.rstu
   and interactive content via web. It teaches R Markdown, Shiny, Tidyverse (dplyr/tidyr/ggplot2/lubridate).
   
 * [Computing for the Social Sciences](http://cfss.uchicago.edu/) University of Chicago; Benjamin Soltoff. This is an applied course for social scientists with little-to-no programming experience who wish to harness growing digital and computational resources. The focus of the course is on generating reproducible research through the use of programming languages and version control software. Major emphasis is placed on a pragmatic understanding of core principles of programming and packaged implementations of methods. Students will leave the course with basic computational skills implemented through many computational methods and approaches to social science; while students will not become expert programmers, they will gain the knowledge of how to adapt and expand these skills as they are presented with new questions, methods, and data.
+
+* [Applied Media Analytics](https://www.elon.edu/u/academics/communications/academics/media-analytics-major/) Elon University; Brian Walsh. An Undergraduate course for Media Analytics majors, students are introduced to R programming via the tidyverse and learn ggplot, dplyr, & lubridate, as well as basic sentiment analysis, Twitter analysis, and Google Analytics insights. 
```

Comment:
I know it's inconsistent across the listings, but I'd suggest maybe breaking this into sentences. "This is an undergraduate course..." And then have a full stop before "Students are introduced to R programming via the tidyverse." Then, maybe rework that last sentence to start with: "Students will learn ggplot**2**, dplyr...".

Let me know if you have any questions.

Reply:
Thanks for the feedback! Will do.